packages/nc-gui/components/virtual-cell/Lookup.vue

```diff
@@ -78,23 +78,23 @@ provide(CellUrlDisableOverlayInj, ref(true))
 const timeout = 3000 // in ms
 
 const showEditWarning = refAutoReset(false, timeout)
+
 const showClearWarning = refAutoReset(false, timeout)
 
 useSelectedCellKeyupListener(inject(ActiveCellInj, ref(false)), (e: KeyboardEvent) => {
   switch (e.key) {
     case 'Enter':
       showEditWarning.value = true
       break
-    case 'Delete':
+    default:
       showClearWarning.value = true
-      break
   }
```

Comment:
since we are setting it to true for any keypress switch case is not at all required here.